automation/src/test/scala/org/broadinstitute/dsde/workbench/leonardo/ClusterMonitoringSpec.scala

```diff
@@ -0,0 +1,60 @@
+package org.broadinstitute.dsde.workbench.leonardo
+
+import org.broadinstitute.dsde.workbench.WebBrowserSpec
+import org.broadinstitute.dsde.workbench.dao.Google.googleIamDAO
+import org.scalatest.{FreeSpec, ParallelTestExecution}
+import org.scalatest.time.{Seconds, Span}
+
+class ClusterMonitoringSpec extends FreeSpec with WebBrowserSpec with LeonardoTestUtils with ParallelTestExecution {
```

Comment:
Minor, looks like `LeonardoTestUtils` already pulls in `WebBrowserSpec` so it's not needed here.